doc/source/_static/custom.css

```diff
@@ -1,16 +1,104 @@
 .sphinx-codeautolink-a{
-    border-bottom-color: rgb(0, 0, 0);
-    border-bottom-style: dotted;
-    border-bottom-width: 1px;
+  border-bottom-color: rgb(0, 0, 0);
+  border-bottom-style: dotted;
+  border-bottom-width: 1px;
 }
 .sphinx-codeautolink-a:hover{
-    color: rgb(255, 139, 139);
+  color: rgb(255, 139, 139);
 }
 
 .supportbutton a {
- color: var(--pst-color-sidebar-link-active);
+color: var(--pst-color-sidebar-link-active);
 }
 
 .navbar-brand {
-  display: inline-block !important;
+display: inline-block !important;
 }
+
+/* -------------------- EXAMPLE GALLERY OVERRIDES -------------------- */
+
+.bd-content .sd-card .sd-card-body,
+.bd-content .sd-card .sd-card-footer {
+  background-color: transparent;
+}
+
+/* Each card has same height (across all rows in the grid) */
+.sd-card.example-gallery {
+  height: 250px;
+  border-radius: 0.2rem;
+  border: 0.6px solid rgba(255,255,255,0) !important;  /* Override Sphinx class */
+}
+
+/* Hover effects for each card */
+.example-gallery:hover {
+  box-shadow: 0 0 0.5rem 0.2rem rgba(15, 187, 188, 0.3) !important;  /* Override Sphinx class */
+  border: 0.6px solid #0fbbbc;
+}
+.example-gallery:hover p {
+  color: #0fbbbc;
+}
```

Comment:
Not sure what are the colors being used and where, I think it's the "culprit" is the `p` one, but commenting over all the style just in case. 

The text in the grid cards is rendered in blue and then switches to a lighter shade of blue when hovering, which I find a bit annoying for the following two reasons:

* neither the base blue nor the lighter "on hover" blue match the link color of the theme, the one in the `edit this page` text for example. I think at least one of the two should be using the theme defined colors: https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/customizing.html#color-variables.
* That behaviour doesn't match the theme behaviour. I have noticed 3 possible cases relative to the theme:
  - Text is black/text base color and becomes blue on hover (navbar and left sidebar behaviour)
  - Text is blue and keeps the same color but becomes underlined on hover (right sidebar behaviour, the donate is custom added and doesn't match the theme either and we should probably fix that too, it should not be considered an example here though)
  - Text is blue and becomes pink and underlined on hover (behaviour on body and toctrees, see https://arviz--2087.org.readthedocs.build/en/2087/user_guide/index.html)

The second point is not that important, the similar yet slightly differnet shades of blue I think should be changed.

Reply:
Thanks for pointing this out. I wasn't exactly sure which color should be used but it's sounds like we should ultimately keep it consistent. 
* [Color] The blue I put in the custom CSS was based on the teal in the ArviZ logo, but since the theme is pydata I could use those predefined colors. Any reason to keep the original theme colors instead of recoloring the blues to fit the ArviZ logo? 
* [Hover] I can take a look at these events and default the example gallery tile hover back to orange underline to keep consistency. LMK if you have a difference preference. I am personally not a huge fan of blue -> orange because it's a bit generic with data visualizations 😂 
* [General] How custom do you want the example gallery and overall website? Can you point me to any inspirations (what aspect you like from other docs) that you would like the docs to look like (or not look like) in a future state?

Reply:
Probably getting distracted, but on the topic of theme/homepage, are you open to different layouts? This mockup below (from pasting screenshots together) uses up real estate more efficiently. And some things I prefer are:
* Title header should go before search bar (So I think ideally it should be H1 > brief description > search bar)
* Example charts can go below. Currently the example charts are shown in the right, but there is nothing else on the right panel. This point is super minor.
* Having more headers (installation, contribution) helps split up the bodies of text! 
* Looking at other data viz documentations, they do not have "Installation/Contribution" details in their homepage even though it seems important here. Instead they have links on where to go ([see bokeh example](https://docs.bokeh.org/en/latest/index.html)). I'm not sure which direction is better... maybe keep what is currently on this page and add a section (before "Installation") on where to start (e.g. example gallery for ideas, QuickStart page, etc.)? 
<img width="496" alt="image" src="https://user-images.githubusercontent.com/20257416/183834379-32521b86-7b31-45fd-8e15-f5a8997d0699.png">

Reply:
> [Color] The blue I put in the custom CSS was based on the teal in the ArviZ logo, but since the theme is pydata I could use those predefined colors. Any reason to keep the original theme colors instead of recoloring the blues to fit the ArviZ logo?

We can also update the predefined colors! I didn't mean that the color from the examples needs to be the one currenly used in the navbar hover, only that they should be the same. 

We can use a custom color scheme defined in our own `custom.css` (or `colors.css`, we can load multiple css files during the doc build). I think we'd only need to follow this: https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/customizing.html#base-variables, then the arviz teal color would be the highlight color for navbar links on hover and so on, but there would not be multiple shades of blue in similar roles.

From what I understand the theme uses two colors mainly, blue and pink in addition to the black/white text (depending on the mode being light or dark) and uses those two for hover effects and highlights and so on. The black to blue is black to primary color I think, the blue to pink is primary to secondary color, I don't really understand the always blue that only becomes underlined, it might not be configurable 🤔 

> [Hover] I can take a look at these events and default the example gallery tile hover back to orange underline to keep consistency. LMK if you have a difference preference. I am personally not a huge fan of blue -> orange because it's a bit generic with data visualizations

We already have at least 3 different hover effects _for text_ all of which indicate clickable links, I am not tied to any of the existing effects, but if possible I'd prefer to not add another hover text effect that also indicates link. 

> [General] How custom do you want the example gallery and overall website? Can you point me to any inspirations (what aspect you like from other docs) that you would like the docs to look like (or not look like) in a future state?

Not really sure how to answer to that, especially the first part. I'd say probably as customizable as possible without taking over a large maintenance burden. That is, we should take advantage of theme customizations and use custom css, we could even create a "theme extension" (like https://github.com/matplotlib/mpl-sphinx-theme) so we can use it in ArviZ, [preliz](https://preliz.readthedocs.io/en/latest/), we could also change the [xarray-einstats](https://einstats.python.arviz.org/en/latest/) one. However, creating our theme from scratch or forking it and renouncing to the improvements that would come with future releases to create a custom "forzen" theme would be a no-go; similarly, I could probably be convinced to add some js but I'd need a lot of convincing.

As for inspirations, here are some gallery pages we could get inspiration from:

* https://matplotlib.org/stable/gallery/index.html
  - Very similar to the current grid style, the images are a bit too small in my opinion, the 3 columns is better, I'll also try to get rid of the right sidebar probably. I'm still on the fence about having text under the image. I do not like the tooltip it has neither in style nor in content. IMO the tooltip should be short and direct, it should not have "this example shows" (the same way alt text should not have "an image of"). I also think it should not be cut half way and use `...`, for the style, on the fence about the tooltip appearing to the right, but I don't like the combination of still needing to read the example title (which is shown in orage over a white background when hovering) together with the tooltip description in white text over a black background.
* https://se.mathworks.com/products/matlab/plot-gallery.html
  - The images are also too little, not sure if still considered a tooltip, but I like the click on the image that generates a pop-up with the image in a larger size.
* http://seaborn.pydata.org/examples/index.html
  - (I think our example gallery was adapted originally from this one, or both were adapted from the same source) I like the no text at all option and perfectly square images, but the cropped square images I think are a case of aesthetics overriding functionality so I would not do/go back to this. But we might want to consider no-text options like here (or more like the example below) and have the text both title and description (as I said I'm partial to bigger images) appear only on hover.
* https://r-graph-gallery.com/density-plot.html
  - I like the style of this gallery, big images and little text, I do not like the hover content nor the hover style though. Given how all examples use viridis colormap, I find the orange to clash too much and also to not have enough contrast with the white text (which becomes even worse because it is also a bit transparent). There is no way this passes the accessibility color contrast checks. You proposes something similar on the issue too, if is is possible to get ti working with colors that allow reading the text easily then we could look into something like this. 
  - Not the thing of the example, but using the "modal style" that https://sphinx-hoverxref.readthedocs.io/en/latest/ supports will probably be simpler to implement (and not depend on us) and also look good, as the "opened" preview would show both description and larger plot, even have working links from the hover window/tooltip.
* https://altair-viz.github.io/gallery/index.html
  - too small images and too small example grid area. This is close to what I don't want the example gallery to look like.

Reply:
> are you open to different layouts?

yes!


>  Title header should go before search bar (So I think ideally it should be H1 > brief description > search bar)

The search bar should be gone. It is currently there because we believe all the pages should have the searchbar accessible somewhere, and it didn't fit in the navbar. The next version of the theme solves that, the search bar will be an icon on the navbar that expands/pops up when clicked on: https://pydata-sphinx-theme.readthedocs.io/en/latest/ (towards the right of the navbar, left of the links)

> Example charts can go below. Currently the example charts are shown in the right, but there is nothing else on the right panel. This point is super minor.

Sounds good, if we put them on a single line they could even be a carrousel: https://sphinx-design.readthedocs.io/en/sbt-theme/cards.html#card-carousels

> Looking at other data viz documentations, they do not have "Installation/Contribution" details in their homepage even though it seems important here. Instead they have links on where to go ([see bokeh example](https://docs.bokeh.org/en/latest/index.html)). I'm not sure which direction is better... maybe keep what is currently on this page and add a section (before "Installation") on where to start (e.g. example gallery for ideas, QuickStart page, etc.)?

We could look into restructuring this. I am also no convinced by the layout nor content but lack ideas on what to change too. 

A general note though. I don't think any text section should be as wide as the navbar. There is a reaso why the theme doesn't take all the available screen and why even if we remove the sidebar it doesn't expand to take all the width by default (see the theme page: https://pydata-sphinx-theme.readthedocs.io/en/latest/, no sidebars yet same width). It makes the text harder to read, bokeh does it and the elevator pitch right above the search bar is a bit hard to follow for me, but we can definitely change the layout. the matplotlib homepage for example: https://matplotlib.org/ is wider than the navbar yet the text sentences are all restricted to at most half the width.